R/wrangle_and_check.R

```diff
@@ -603,6 +604,20 @@ check_results <- function(wrangled_results_list, sensitivity_analysis_vars) {
       name_data = "Company Trajectories"
     )
 
+  # crispy results ----------------------------------------------------
+  wrangled_results_list$crispy_output %>%
+    report_missings(
+      name_data = "CRISPY Results"
+    ) %>%
+    report_all_duplicate_kinds(
+      composite_unique_cols = c(
```

Comment:
please move to lookup file

Reply:
just for my understanding, we should move it for crispy, but not for the other outputs, because crispy has a fixed set of cuc variables and the others do not?

Reply:
oh, i thought we had nicely put all of them into lookup, will close comment then